lighthouse-core/test/results/sample_v2.json

```diff
@@ -5461,6 +5461,18 @@
         "duration": 100,
         "entryType": "measure"
       },
+      {
+        "startTime": 0,
+        "name": "lh:computed:MainResource",
+        "duration": 100,
+        "entryType": "measure"
+      },
+      {
+        "startTime": 0,
+        "name": "lh:computed:PageDependencyGraph",
```

Comment:
if these are here it means the computed artifact caching isn't working, you might need to double check that you're passing in just the set of actual required artifacts to each